docs/core/dotnet-five.md

```diff
@@ -0,0 +1,161 @@
+  - [Many more areas](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-5)
+- Container size optimizations
+- [App trimming](https://devblogs.microsoft.com/dotnet/app-trimming-in-net-5)
+- [C# compiler enhancements](https://devblogs.microsoft.com/dotnet/automatically-find-latent-bugs-in-your-code-with-net-5)
+- Tooling support for dump debugging
+- Platform is 80% annotated for [nullable reference types](csharp/nullable-references.md)
+
+### What .NET 5 is not
+
+.NET 5 is not a replacement for .NET Framework. There are no plans to port the following technologies from .NET Framework to .NET 5, but there are supported alternatives included in .NET 5:
+
+| Technology                             | Recommendation                                              |
+|----------------------------------------|-------------------------------------------------------------|
+| Web Forms                              | [ASP.NET Core Blazor](/aspnet/core/blazor)                  |
+| Windows Communication Foundation (WCF) | [gRPC](/aspnet/core/grpc)                                   |
+| Windows Workflow (WF)                  | [Open-source CoreWF](https://github.com/UiPath-Open/corewf) |
+
+## .NET Standard
+
+New application development can specify the `net5.0` target framework moniker (TFM) for all project types, including class libraries. Sharing code between .NET 5 workloads is simplified in that all you need is the `net5.0` TFM.
+
+The `net5.0` TFM combines and replaces `netcoreapp` and `netstandard` names. This TFM will generally only include technologies that work cross-platform, like was done with .NET Standard. However, if you're planning on sharing code between .NET Framework, .NET Core, and .NET 5 workloads - you can do so by specifying `netstandard2.0` as your TFM. For more information, see [How to specify target frameworks](standard/frameworks.md#how-to-specify-target-frameworks).
+
+## Language updates
+
+With .NET 5, the .NET programming languages are continuing to improve.
+
+### C# updates
+
+Developers writing .NET 5 apps will have access to the latest C# version and features. .NET 5 is paired with C# 9. C# 9 brings many new features to the language, here are a few highlights:
+
+- Records: Immutable reference types that behave like value types, and introduce the new `with` keyword into the language.
+- Relational pattern matching: Extends pattern matching capabilities to relational operators for comparative evaluations and expressions, including logical patterns - new keywords `and`, `or`, and `not`.
+- Top-level statements: As a means for accelerating adoption and learning of C#, the `Main` method can be omitted and application as simple as the following is valid:
+
+   ```csharp
+   System.Console.Write("Hello world!");
+   ```
+
+- Functional pointers: Language constructs that expose intermediate language (IL) the following opcodes `ldftn` and `calli`.
```

Comment:
@IEvangelist Should this read like the following?

>  - Functional pointers: Language constructs that expose the following intermediate language (IL) opcodes `ldftn` and `calli`.

Reply:
I'd approve and merge that PR, if you want to propose the change - thanks @Youssef1313